src/proto_conversions.cc

```diff
@@ -0,0 +1,215 @@
+#include "proto_conversions.h"
+
+namespace valhalla {
+
+bool Options_Action_Enum_Parse(const std::string& action, Options::Action* a) {
```

Comment:
since these are indeed proto conversions, i moved them out of worker.h/cc and into the proto_conversions.h/cc